lib/v2/qdu/maintainer.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
     '/jwc': ['abc1763613206'],
+    '/houqin': ['abc1763613206'],
```

Comment:
Insert new entries in alphabetical order.
```suggestion
    '/houqin': ['abc1763613206'],
    '/jwc': ['abc1763613206'],
```